.github/actions/sdk-codegen/entrypoint.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+set -e
+
+function help() {
+  echo "Entry point of our sdk code gen container."
+  echo "-k|--sdk             - Which sdk to generate (GO,JS,JAVA,ALL)"
+  exit
+}
+
+PARAMS=""
+
+while (( "$#" )); do
+  case "$1" in
+    -k|--sdk)
+      shift
+      SDK=$1
+      ;;
+    -h)
+      help
+      ;;
+    -*|--*=) # unsupported flags
+      echo "Error: Unsupported flag $1" >&2
+      exit 1
+      ;;
+    *) # preserve positional arguments
+      PARAMS="$PARAMS $1"
+      ;;
+  esac
+  shift
+done
+# set positional arguments in their proper place
+eval set -- "$PARAMS"
```

Comment:
nit: positional params are not used, may as well let them generate an error.
```suggestion
    *) # unsupported flags
      echo "Error: Unsupported flag $1" >&2
      exit 1
      ;;
  esac
  shift
done
```